src/app/action-items/action-items.component.html

```diff
@@ -36,12 +37,18 @@
                 </div>
                 <div class="flexRow bodyCell inner" role="cell">{{ t.type }}</div>
                 <div class="flexRow bodyCell inner" role="cell">{{ t.completed }}%</div>
-                <div class="flexRow bodyCell inner" role="cell">{{ (t.dueDate | date: 'dd/MM/yyyy') || '-' }}</div>
-                <div class="flexRow bodyCell inner" role="cell">{{ t.dueDay }}</div>
+                <div class="flexRow bodyCell inner" role="cell">{{ (t.dueDate | date: 'MM/dd/yyyy') || '-' }}</div>
+                <div class="flexRow bodyCell inner" role="cell">
+                    {{ t | daysLeftCounted }}
+                </div>
+                <div class="flexRow bodyCell inner" role="cell">
+                    <a><i class="fa fa-edit"></i></a>
+                </div>
             </div>
         </div>
     </div>
-    <div class="innerContainer detailsContainer">
-        <app-item-details></app-item-details>
+    <div class="innerContainer detailsContainer ">
+        <app-item-details [id]="actionItemId" [item]="actionItem" [isLoadingActionItem]="isLoadingActionItem" [currentDate]="currentDate" [errorMessage]="errorMessage">
```

Comment:
`currentDate` is not defined in the component so you are always passing undefined here. Why?